src/Simulation/Simulators/QuantumSimulator/QuantumSimulator.cs

```diff
@@ -93,6 +93,8 @@ static void CheckAngle(double angle)
         void CheckQubit(Qubit q1)
```

Comment:
For the methods in this file that set `IsMeasured` to `false`, could you add to their summaries that they do this?